modules/programs/taskwarrior.nix

```diff
@@ -88,7 +88,7 @@ in {
   config = mkIf cfg.enable {
     home.packages = [ pkgs.taskwarrior ];
 
-    xdg.configFile."task/taskrc".text = ''
+    xdg.configFile."task/taskrc_".text = ''
```

Comment:
I wasn't even sure if the rename is needed. The website doesn't mention anything about reading a file from `~/.config`. "Better safe than sorry".